beacon-chain/core/blocks/block_operations.go

```diff
@@ -35,29 +35,6 @@ var log = logrus.WithField("prefix", "blocks")
 
 var eth1DataCache = cache.NewEth1DataVoteCache()
 
-// ErrSigFailedToVerify returns when a signature of a block object(ie attestation, slashing, exit... etc)
-// failed to verify.
-var ErrSigFailedToVerify = errors.New("signature did not verify")
-
-func verifySigningRoot(obj interface{}, pub []byte, signature []byte, domain []byte) error {
```

Comment:
Moved this to /core/helpers/signing_root.go

Not sure why it was here to begin with....